server_properties.yaml

```diff
@@ -1,5 +1,5 @@
 port: 25565
 ip: 0.0.0.0
```

Comment:
dont change this. not only is this not relevant to this PR, but we are aiming for it to be as vanilla as possible.

Reply:
I don't really think that it matters, does it?

Reply:
Looks like the comment is on the wrong line

Reply:
I meant the default server motd